packages/api/src/providers/fitbit.ts

```diff
@@ -332,4 +336,75 @@ export class Fitbit extends Provider implements OAuth2 {
       }
     );
   }
+
+  async postAuth(token: string, userId?: string) {
+    const accessToken = JSON.parse(token).access_token;
+
+    const scopes = JSON.parse(token).scope;
+
+    const subscriptionTypes: Record<FitbitScopes, string> = {
+      [FitbitScopes.activity]: "activities",
+      [FitbitScopes.nutrition]: "foods",
+      [FitbitScopes.weight]: "body",
+      [FitbitScopes.sleep]: "sleep",
+    };
+
+    const subscriptionId = userId;
+
+    // TODO: Part of the TODO just below. This code could be used to subscribe to all collectionTypes, including "userRevokedAccess"
+    // if (this.allRequiredScopesIncluded(scopes)) {
+    //   const subscriptionUrl = `${Fitbit.URL}/${Fitbit.API_PATH}/apiSubscriptions/${subscriptionId}.json`;
+    //   this.createSubscription(subscriptionUrl, accessToken);
+    //   console.log("All scopes included. User subscribed to all Fitbit WH collectionTypes.");
+    // } else {
```

Comment:
Left-over comments, here and below after the `Promise.all`?

Reply:
Gotcha.

Just so we don't leave outdated code lingering (hard to remove later), are you planning to tackle #652 soon? If not, could copy/paste the code on the issue as a reference and keep the codebase clean.

Reply:
I would imagine this should be something to handle soon after most other tasks in the current backlog are done.

Reply:
I'd move this to the ticket so we have the code clean and not tied to priorities that might change. But not to big of a deal.